frontend/src/components/common/HashTag/HashTag.tsx

```diff
@@ -13,7 +13,13 @@ const HashTag = ({ hashTags, setHashTags }: HashTagProps) => {
 	const [hashTagInput, setHashInput] = useState<string>('');
 	const [errorMsg, setErrorMsg] = useState<string>('');
 
-	const checkIsDuplicateHashTag = () => hashTags.includes(hashTagInput);
+	const checkIsDuplicateHashTag = (hashtag: string) =>
+		hashTags.reduce((acc, cur) => {
+			if (cur.toLowerCase() === hashtag.toLowerCase()) {
+				acc = true;
+			}
+			return acc;
+		}, false);
```

Comment:
중복처리하는 함수인것 같은데
```suggestion
	const checkIsDuplicateHashTag = (hashtag: string) => {
	   const checkSet = new Set(hashTags.map(hashTag => hashTag.toLowerCase()));
	   const prevLength = checkSet.size;
	   checkSet.add(hashtag.toLowerCase());
	   
	   return prevLength === checkSet.size;
	}
```
이런식의 변경도 가능하지 않을까요??

Reply:
엄.. 가능은 하겠지만 바꿀 이유를 잘 모르겠네요 reduce는 이럴때 쓸려고 사용한다고 개인적으로 생각해서

Reply:
엄... 가능은 하겠지만 굳이도 싶네요? 개인적으로 reduce로 이런거 처리하는 게 좋을 것 같아서요

Reply:
여기서는 큰 문제는 없겠지만 불필요한 시간복잡도를 이용해서 처리하는것 같아서 Set을 추천했습니다